modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache

```diff
@@ -133,12 +133,18 @@ export const {{classname}}AxiosParamCreator = function (configuration?: Configur
     {{#headerParams}}
             {{#isListContainer}}
             if ({{paramName}}) {
-                localVarHeaderParameter['{{baseName}}'] = {{paramName}}.join(COLLECTION_FORMATS.{{collectionFormat}});
+                let mapped = {{paramName}}.map(value => (<any>"{{{dataType}}}" !== "Array<string>") ? JSON.stringify(value || {}) : (value || ""));
```

Comment:
if we have an `Array<number>` with `[0, 2, 3]`, wouldn't this be serialized to `{},2,3`?

Reply:
maybe, do you have a suggested fix?

Reply:
did your suggestion fix this too? Im not sure right now, so many months have passed ;)

Reply:
yes, this should be fixed with my suggestion